cfme/intelligence/chargeback/assignments.py

```diff
@@ -75,27 +77,21 @@ class Assign(Updateable, Pretty, Navigatable):
     enterprise.computeassign()
 
     """
-    def __init__(self, assign_to=None,
-                 tag_category=None,
-                 docker_labels=None,
-                 selections=None,
-                 appliance=None):
-        Navigatable.__init__(self, appliance=appliance)
-        self.assign_to = assign_to
-        self.tag_category = tag_category
-        self.docker_labels = docker_labels
-        self.selections = selections
+    assign_to = attr.ib(default=None)
+    tag_category = attr.ib(default=None)
+    docker_labels = attr.ib(default=None)
+    selections = attr.ib(default=None)
 
     def storageassign(self):
-        view = navigate_to(self, 'Storage')
+        view = navigate_to(self, "Storage")
```

Comment:
Required: Please revert " to ', update all places

Reply:
Thanks for the review, I'll make the changes asap. Actually this conversion of  ' to " is something that I picked up from https://github.com/ManageIQ/integration_tests/pull/7526/files :) . However since it is not needed, I'll make sure that I revert it back